uberjar/src/test/java/io/fabric8/kubernetes/clnt/UberJarTest.java

```diff
@@ -159,7 +159,8 @@ private String getMinorVersion(String projectVersion) {
   }
 
   private File getFileInDirectory(File parentFile, String pathToFile) {
-    pathToFile = pathToFile.replaceAll("/", File.separator);
+    String toReplaceChar = File.separator.equals("/") ? File.separator : "\\\\";
```

Comment:
This seems unnecessary, why is it needed?

Also, a simpler:
```java
pathToFile = pathToFile.replaceAll('/', File.separatorChar);
```
 should work

Reply:
Without this change this test was failing on windows with this error `java.lang.IllegalArgumentException: character to be escaped is missing` 

I found this link https://bugs.openjdk.java.net/browse/JDK-8165200 and updated test accordingly. Let me try your suggestion.